### DISCUSSION TITLE
Save column filters in the grid configuration

### DISCUSSION THREAD

**Discussion by alexdrupal** [pimcore/pimcore#15304] [Q&A / Support]
Hello

I've noticed that if I use grid search (data object folder) and then save the grid configuration - it saves search query as well. Same thing but with column filters doesn't work.

Example: I have a column - SKU and enter a particular SKU number. So the result grid shows only one row. Now I would like to save this grid configuration so that next time I can use it and have 1 row displayed.

Is it possible already or needs some dev work to implement?

**Comment by SamyMP** [comment]
It is possible if you use Pimcore 11.x, [see this PR](https://github.com/pimcore/pimcore/pull/13498)
